### DISCUSSION TITLE
Can't send more than 1481878781843 in a single transaction,How to solve it?

### DISCUSSION THREAD

**Discussion by wmbbt** [Chia-Network/chia-blockchain#8320] [Support]
Can't send more than 1481878781843 in a single transaction,How to solve it?

**Comment by loppefaaret** [comment]
i think it maxes out the cion ID's in the attempt to send more - either mint the coins into bigger coin ID's, or send multiple transactions

**Comment by BrandtH22** [comment]
@wmbbt , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
